pom.xml

```diff
@@ -27,13 +27,14 @@
 
 
   <properties>
-    <spark.version>3.4.1</spark.version>
+    <spark.version>3.5.5</spark.version>
     <scala.binary.version>2.12</scala.binary.version>
-    <fasterxml.jackson.version>2.14.3</fasterxml.jackson.version>
+    <fasterxml.jackson.version>2.18.3</fasterxml.jackson.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <aws.sdkv2.version>2.23.9</aws.sdkv2.version>
-    <aws.kpl.version>0.15.8</aws.kpl.version>
+    <aws.sdkv2.version>2.31.11</aws.sdkv2.version>
+    <aws.kpl.version>0.15.12</aws.kpl.version>
+    <log4j.version>2.24.3</log4j.version>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
nit: These are all properties, right? Maybe we should indent consistently now or in a future PR.

Reply:
Correct, these are all properties that define dependency versions.

Reply:
Ignore this. This was my problem with word-wrapping in the web UI.